specification/dimensions/resourcetype.md

```diff
@@ -0,0 +1,53 @@
+# Resource Type
+
+A Resource ID is an identifier assigned to a resource by the provider. The Resource ID is commonly used for cost reporting, analysis, and allocation scenarios.  While Resource ID and Resource Name identifies a unique instance of a resource, the Resource Type describes the identity of the resource. Resource Type is commonly used for cost reporting, analysis, and allocation scenarios.
+
+The ResourceType column MUST be present within billing data.  ResourceType MUST be of type String and MUST NOT be NULL when a corresponding ResourceId is not NULL.  When a corresponding ResourceId value is NULL, the ResourceType column value MUST be NULL.
+
+## Column ID
+
+ResourceType
+
+## Display Name
+
+Resource Type
+
+## Description
+
+Classifiable description of a corresponding ResourceId
```

Comment:
We usually make this the same as the intro paragraph (minus scenarios). I like this description tho. Would be good to work this into the one above. Maybe use "Resource ID" in this context.

Reply:
Edit above and duplicated here, thoughts?
```
A classifiable representation of a specific kind of cloud service or corresponding resource that can be provisioned, configured, and managed by users.
```